src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1576,6 +1576,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help='An opaque continuation token. This value can be retrieved from the next_marker field of a '
                    'previous generator object. If specified, this generator will begin returning results from this '
                    'point.')
+        c.argument('show_next_marker', action='store_true',
```

Comment:
could you add preview for the flag in this sprint?

Reply:
Sure. Done~